sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -98,6 +98,20 @@ export const studentRoutes: Array<RouteRecordRaw> = [
           clientType: ClientIdType.Student,
         },
       },
+      {
+        path: AppRoutes.StudentApplicationChangeRequest,
+        name: StudentRoutesConst.DYNAMIC_FINANCIAL_CHANGE_REQUEST_APP_FORM,
+        component: DynamicStudentApp,
+        props: (route: RouteLocationNormalized) => ({
+          id: Number(route.params.id),
+          programYearId: Number(route.params.programYearId),
+          selectedForm: route.params.selectedForm as string,
+          changeRequest: true,
```

Comment:
is `changeRequest` a param in the route?

Reply:
It is not a parameter in the URL, it is a parameter provided to the component associated with the route.